### ISSUE TITLE
ImportError: cannot import name 'RegexFlag' from 'regex' 

### ISSUE THREAD

**Issue by m-ar-c** [rrthomas/rpl#18]
Hi. I just updated rpl from this repo today. Now I have the following error :

``` /opt/rpl/rpl --version
# rpl --version
Traceback (most recent call last):
  File "/opt/rpl/rpl", line 35, in <module>
    from regex import RegexFlag
ImportError: cannot import name 'RegexFlag' from 'regex' (/usr/lib/python3/dist-packages/regex/__init__.py)
```

The command is run inside a debian 11.5 lxc container.

Do I need to install a new package on the system ?

**Comment by rrthomas** [comment]
Apologies, it seems that `RegexFlag` was introduced in version 2022.3.2. I will add a suitable version dependency.

**Comment by rrthomas** [comment]
I've now released 1.15rc2 with the fix.

**Comment by m-ar-c** [comment]
Thanks!
